scalabel/label/io.py

```diff
@@ -21,6 +21,9 @@ def parse(raw_frame: DictStrAny) -> Frame:
 def load(inputs: str, nprocs: int = 0) -> List[Frame]:
     """Load labels from a json file or a folder of json files."""
     raw_frames: List[DictStrAny] = []
+    if not osp.exists(inputs):
```

Comment:
Is this line really needed? Consider the logics from L27 to L44, the new added line seems a bit extra

Reply:
if i integrate it into the elif cascade, pylint complains with "Unnecessary "elif" after "raise" (no-else-raise)".
I think this can be viewed as a pre-condition, which has to be fulfilled for the rest to make sense.

Reply:
Understand, sure~